Improvement in the documentation (#637)
apache/openwhisk-deploy-kube: README.md

You can use a different name and/or namespace simply by changing the commands
used below.

> **NOTE:** Clone the repository https://github.com/apache/openwhisk-deploy-kube.git and use to helm chart available under the `helm/openwhisk` folder.

Reviewer: ```suggestion
**NOTE:** Clone the repository https://github.com/apache/openwhisk-deploy-kube.git and use to Helm chart available under the `helm/openwhisk` folder.
```

Deployment can be done by using the following single command:
```shell
helm install owdev ./helm/openwhisk -n openwhisk --create-namespace -f mycluster.yaml
```

> **NOTE:** The above command will only work for Helm v3.2.0 or higher versions. Verfiy your local helm version with command `helm version`.

Reviewer: ```suggestion
**NOTE:** The above command will only work for Helm v3.2.0 or higher versions. Verfiy your local Helm version with the command `helm version`.
```

You can use the command `helm status owdev -n openwhisk` to get a summary
of the various Kubernetes artifacts that make up your OpenWhisk
deployment. Once the pod name containing the word `install-packages` is in the `Completed` state,
your OpenWhisk deployment is ready to be used.

> **NOTE:** You can check the status of the pod by running the following command `kubectl get pods -n openwhisk --watch`

Reviewer: ```suggestion
**NOTE:** You can check the status of the pod by running the following command `kubectl get pods -n openwhisk --watch`.
```

 -Dwhisk.auth=$WHISK_AUTH -Dwhisk.server=https://$WHISK_SERVER -Dopenwhisk.home=`pwd`
```

> **NOTE:** You to install JDK 8 in order to run these test case

Reviewer: ```suggestion
**NOTE:** You need to install JDK 8 in order to run these tests.
```
